src/backend/distributed/executor/executor.c

```diff
@@ -532,8 +544,6 @@ StartDistributedExecution(DistributedExecution *execution)
 	{
 		RecordParallelRelationAccessForTaskList(taskList);
 	}
-
-	AssignTasksToConnections(execution);
```

Comment:
Now that we have remove this line, we should update comments for `StartDistributedExecution()`.